src/libServer/LookupServer.cpp

```diff
@@ -792,6 +847,187 @@ Json::Value LookupServer::CreateTransaction(
   }
 }
 
+Json::Value LookupServer::CreateTransactionEth(EthFields const& fields, bytes const& pubKey, const unsigned int num_shards,
```

Comment:
Is it neccassary to have this code in both isolated server and lookup server, could it not be common to both? just asking :-) or is this just temporary ?

Reply:
the code that can be common is common - so the isolated server calls the lookup server's implementation when it is simple (like getting the chain id)